tidb-binlog-deployment-topology.md

```diff
@@ -28,14 +28,22 @@ aliases: ['/docs-cn/dev/tidb-binlog-deployment-topology/']
 
 拓扑配置模版的关键参数如下：
 
-- `binlog.enable: true`
+- `server_configs.tidb.binlog.enable: true`
 
     开启 binlog 服务，默认为 false。
 
-- `binlog.ignore-error: true`
+- `server_configs.tidb.binlog.ignore-error: true`
 
     高可用场景建议开启，如果设置为 true，发生错误时，TiDB 会停止写入 binlog，并且在监控项 `tidb_server_critical_error_total` 上计数加 1；如果设置为 false，一旦写入 binlog 失败，会停止整个 TiDB 的服务。
 
+- `drainer_servers.config.syncer.db-type`
+
+    binlog 的下游类型，目前支持 `mysql`，`tidb`，`kafka`，`file`。
```

Comment:
Could you please add some examples for them?

Reply:
done in https://github.com/pingcap/docs-cn/pull/5049/commits/82b0f95b2ed0adecbde3aea9fd21898e96d2f398

Reply:
Thx.